projects/cardiovascular-patient-app/src/app/app.component.html

```diff
@@ -1,10 +1,11 @@
 <mat-toolbar>
   <span>{{ appTitle }}</span>
 </mat-toolbar>
-<mat-tab-group mat-stretch-tabs="false" mat-align-tabs="start">
+<mat-tab-group mat-stretch-tabs="false" mat-align-tabs="start" [selectedIndex]="selectedIndex"
+  (selectedIndexChange)="selectedIndex = $event">
   <mat-tab label="Report BP">
     <last-report-bp></last-report-bp>
-    <report-bp></report-bp>
+    <report-bp (resourceCreated)="selectedIndex = $event"></report-bp>
```

Comment:
This should not be using `$event` as a tab index. The child component that emits this event does not know which tab should be selected. It doesn't know anything about the tabs.

Reply:
@semicolin child component that emits this event with value 1 means the second tab is selected because for every tab there is an index. and we are emitting.

Reply:
ReportBPComponent should not be telling the AppComponent which tab to select. Keeping track of which tab needs to be selected is AppComponent's responsibility. What if we add an instance of report-bp component that is not inside a tab? Then its output would make no sense. What if we add another tab or change their order? Then we would need to update the report-bp component and make its output emit a different number, even though we did not change anything about that component.